src/libraries/System.IO.FileSystem/tests/System.IO.FileSystem.Tests.csproj

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <IncludeRemoteExecutor>true</IncludeRemoteExecutor>
-    <TargetFrameworks>$(NetCoreAppCurrent)-windows;$(NetCoreAppCurrent)-Unix;$(NetCoreAppCurrent)-Browser</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent)-windows;$(NetCoreAppCurrent)-Unix;$(NetCoreAppCurrent)-Browser;$(NetCoreAppCurrent)-OSX;$(NetCoreAppCurrent)-iOS;$(NetCoreAppCurrent)-tvOS;$(NetCoreAppCurrent)-MacCatalyst</TargetFrameworks>
```

Comment:
Why is this needed?

Reply:
It was added in one of the earliest commits. Before the change, the tests were always failing on macOS because it was using the common unix implementation rather than the macOS implementation.

Reply:
I ran the tests locally on Mac with this change undone and all finished fine. Can you please send a commit undoing this in order to confirm if it is no longer required?

Reply:
I will put this in the next commit.